src/util/math.hpp

```diff
@@ -9,6 +9,17 @@ namespace big::math
 		return (float)radian;
 	}
 
+	inline int meters_to_feet(int meters)
+	{
+		float conversion_rate = 3.28084;
+		return (int)(meters * conversion_rate);
+	}
+
+	inline int divided_by_ten(int feet)
+	{
+		return (int)(feet / 10);
+	}
```

Comment:
Seems a bit overkill to move divide by 10 here.

Should basically be coded right in where you need it.